doc/releases/changelog-dev.md

```diff
@@ -71,34 +71,12 @@
 added `binary_mapping()` function to map `BoseWord` and `BoseSentence` to qubit operators, using standard-binary mapping.
   [(#6564)](https://github.com/PennyLaneAI/pennylane/pull/6564)
 
-
-<h4>New API for Qubit Mixed</h4>
```

Comment:
This ended up in-between most of "new features" and the rest of "new features" - here its just moved further down